ember-inflector/package.json

```diff
@@ -9,8 +9,14 @@
   "license": "MIT",
   "author": "",
   "exports": {
-    ".": "./dist/index.js",
-    "./*": "./dist/*.js",
+    ".": {
+      "types": "./declarations/index.d.ts",
```

Comment:
Why not point these at the original src d.ts?

Reply:
in this case we need to add in package.json under files also `src` or? and so we do publish to much files or?

Reply:
adding `src` under `files` would be sufficient (and common for manual .d.ts)

Reply:
changes... is it so correct?

Reply:
ye